R/Connection.R

```diff
@@ -647,16 +717,90 @@ odbcListDrivers <- function(keep = getOption("odbc.drivers_keep"), filter = getO
 
 #' List Available Data Source Names
 #'
-#' List the available data sources on your system. See the [DSN Configuration
-#' files](https://github.com/r-dbi/odbc#dsn-configuration-files) section of the
-#' package README for details on how to install data sources for the most
-#' common databases.
+#' @description
+#'
+#' Collect information about the available data source names (DSNs).
 #'
-#' @return A data frame with two columns.
+#' @return A data frame with two columns:
 #' \describe{
-#'   \item{name}{Name of the data source}
-#'   \item{description}{Data Source description}
+#'   \item{name}{Name of the data source. The entries in this column can be
+#'     passed to the `dsn` argument of [dbConnect()].}
+#'   \item{description}{Data source description.}
 #' }
+#'
+#' @section Configuration:
+#'
+#' This function interfaces with the driver manager to collect information
+#' about the available data source names.
+#'
+#' For **MacOS and Linux**, the odbc package supports the unixODBC driver
+#' manager. unixODBC looks to the `odbc.ini` _configuration file_ for information
+#' on DSNs. Install unixODBC with `brew install unixodbc`, and find the
+#' location(s) of your `odbc.ini` file(s) with `odbcinst -j`.
```

Comment:
I think you can drop this here and we'll document centrally.

Reply:
Where is the central location you have in mind? My thought was that `odbcListDataSources()` and `odbcListDrivers()` are the functions that most closely interface with the `.ini` files and thus could be those central locations that we link to elsewhere. The "most" central places, e.g. README and `?odbc()`, can document the mental model only and refer here for technicalities.

Reply:
The note on unixODBC installation dropped in [c99a80b](https://github.com/r-dbi/odbc/pull/649/commits/c99a80b2dd361ccecc5bbce11c63be7fc5f3e514).

Reply:
I was thinking we'd have a new function like `odbcListConfig()` or something

Reply:
Is this comment in reference to the whole Configuration section or just the paragraph above the line mentioned? If the latter, removing the paragraph would make the reference to `odbcinst.ini` in the following line come out of nowhere. Maybe we wait until we have some other docs to point to before paring this back further?